### ISSUE TITLE
Update skewed line location to improve line detection

### ISSUE THREAD

**Issue by kolhizin** [kolhizin/bujo_poc#35]
Fine-tune local point location based on loss function of distance to 1-points

### Algorithm
1. Search continuous region with highest density
2. Go in both directions from this region
3. Tweak whole remainder of line to find local maxima
4. Continue to go towards ends tweaking line

**Comment by kolhizin** [comment]
With high enough threshold jumps to another line. With low enough does nothing. Local approach does not work.
